app/templates/manage_communications.html

```diff
@@ -0,0 +1,106 @@
+{% import 'macros/toolkit_forms.html' as forms %}
+
+{% extends "_base_page.html" %}
+
+{% block content %}
+{%
+    with items = [
+        {
+            "link": url_for('.index'),
+            "label": "Admin home"
+        }
+    ]
+%}
+{% include "toolkit/breadcrumb.html" %}
+{% endwith %}
+
+<div class="page-container">
+{% with messages = get_flashed_messages(with_categories=true) %}
+{% if messages %}
+{% for category, message in messages %}
+{% if category == 'upload_communication' %}
+  <div class="banner-success-without-action">
+    <p class="banner-message">
+    {% if message == 'itt_pack' %}
+      ITT pack was uploaded.
+    {% elif message == 'clarification' %}
+      New clarification was uploaded.
+    {% elif message == 'communication' %}
+      New communication was uploaded.
+    {% endif %}
+    </p>
+  </div>
+{% elif category == 'not_pdf' %}
```

Comment:
These `not_*` messages should use [the validation masthead pattern](http://alphagov.github.io/digitalmarketplace-frontend-toolkit/forms/validation.html), linking to the appropriate form controls below.